app/src/components/Scheduling/ContactReview.js

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect, useState } from 'react';
+import Header from './Header';
+import {
+  TextField,
+  FormControl,
+  Button,
+} from '../../store/index';
+import { useStyles } from './ContactReviewStyles';
+
+const HARD_CODED_DATA = {
+  caseWorker: 'Tracy',
+  firstName: 'Nina',
+  lastName: 'Mosley',
+  email: 'nina99@gmail.com',
+  phoneNumber: '+1 (213) 555-5555'
+}
+
+export default function ContactReview() {
+  const classes = useStyles();
+  const [inputs, setInputs] = useState({
+    caseWorker: '',
+    firstName: '',
+    lastName: '',
+    email: '',
+    phoneNumber: '',
+  })
+
+  const [editMode, toggleEdit] = useState({
+    email: false,
+    phoneNumber: false,
+  });
+
+  useEffect(() => {
+    setInputs(HARD_CODED_DATA);
+  }, [])
+
+  const handleChange = (e) => {
+    setInputs({...inputs, [e.target.name]: e.target.value})
+  };
+
+  return (
+    <div className={classes.root}>
+      <Header
+        title="Contact Information"
+        description="Review your profile information below"
+        activity={`Training with ${HARD_CODED_DATA.caseWorker}`}
+      />
+
+      <FormControl>
+        <>
```

Comment:
Curious what your thinking was behind using the fragments (`<></>`) within the return statement. They don't have any stylistic or semantic meaning, and they are not rendered to the page. https://reactjs.org/docs/fragments.html

Because this component's return statement is wrapped in a <div>, you the fragments are unneeded.

Reply:
I most definitely used fragments incorrectly. My thinking at the time was to separate them. But that was entirely wrong after looking at the docs. I removed the fragments.